src/pgen/disk.hpp

```diff
@@ -814,11 +814,21 @@ void DiskBoundaryExtrap(std::shared_ptr<MeshBlockData<Real>> &mbd, bool coarse)
       });
 }
 
+//----------------------------------------------------------------------------------------
+//! \fn TaskStatus ProblemGeneratorSourceTerm()
+//! \brief Custom source term for disk pgen
+template <Coordinates GEOM>
+TaskStatus ProblemGeneratorSourceTerm(MeshData<Real> *md, const Real time,
+                                      const Real dt) {
+  PARTHENON_FAIL("Disk pgen-defined source term not yet implemented!");
+  return TaskStatus::complete;
+}
```

Comment:
Is there a reason why you put a stub in this pgen and not all others? I'm not sure it's needed since it's clear from the `std::function` in `problem_modifier` what the signature is.

Reply:
Just as a demonstration.  Notice that we also use the disk pgen to hold a stub for user-defined refinement criteria.